docs/tools/vdb_table/data/azureai.json

```diff
@@ -119,11 +121,11 @@
   },
   "multi_tenancy": {
     "support": "full",
-    "source_url": "https://learn.microsoft.com/en-us/azure/search/search-modeling-multitenant-saas-applications",
-    "comment": "https://learn.microsoft.com/en-us/azure/search/search-modeling-multitenant-saas-applications"
+    "source_url": "https://learn.microsoft.com/azure/search/search-modeling-multitenant-saas-applications",
+    "comment": ""
   },
   "disk_index": {
-    "support": "",
+    "support": "none",
```

Comment:
Indexes *are* stored and managed on disk (reading the description from the published table), so shouldn't we say "full" to this? I understood that you'd say none if it's an in-memory solution with no storage support.